pkgs/tools/package-management/wapm/cli/default.nix

```
@@ -10,19 +11,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "wapm-cli";
-  version = "0.5.5";
+  version = "0.5.6";
 
   src = fetchFromGitHub {
     owner = "wasmerio";
     repo = "wapm-cli";
     rev = "v${version}";
-    sha256 = "sha256-BKBd1tJwV4VOjRnAx/spQy3LIXzujrO2SS5eA1uybNA=";
+    sha256 = "sha256-QgQQ0lbr7Ggd2HmKlu/5TMFISxrwKWstoAq8e776l8I=";
   };
 
-  cargoSha256 = "sha256-dv04AXOnzizjq/qx3qy524ylQHgE4gIBgeYI+2IRTug=";
+  cargoSha256 = "sha256-F4BvP1yMJ06QgocD6InwkfHtJIsEkOfxuaaalksJCew=";
 
   buildInputs = [ libiconv openssl ]
```

```suggestion
  buildInputs = lib.optionals stdenv.isLinux [ openssl pkg-config ]
```
or is openssl used on darwin, too? That is not always the case.